packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -682,7 +689,7 @@ export class BaseResolversVisitor<
       allResolversTypeName: getConfigValue(rawConfig.allResolversTypeName, 'Resolvers'),
       rootValueType: parseMapper(rawConfig.rootValueType || '{}', 'RootValueType'),
       namespacedImportName: getConfigValue(rawConfig.namespacedImportName, ''),
-      avoidOptionals: getConfigValue(rawConfig.avoidOptionals, false),
+      avoidOptionals: normalizeAvoidOptionals(rawConfig.avoidOptionals),
```

Comment:
This plugin seems to be the only one that's not using `normalizeAvoidOptionals`. This makes it consistent.